components/compliance-service/integration_test/reporting_server_export_test.go

```diff
@@ -136,10 +136,8 @@ func TestReportingServerExport(t *testing.T) {
 
 	for _, test := range successCases {
 		t.Run(test.description, func(t *testing.T) {
-			response, err := client.Export(ctx, &reporting.Query{
-				Filters: []*reporting.ListFilter{{Type: "projects", Values: test.allowedProjects}},
-				Type:    "json",
-			})
+			ctx := auth_context.NewOutgoingContext(contextWithProjects(test.allowedProjects))
```

Comment:
This was the other necessary change, because this test more closely simulates an actual request it needs to prepare the context like an actual network request would.